### DISCUSSION TITLE
Obstacle avoidance using Ardupilot in Airsim

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/AirSim#4874] [Support Q&A]
With the sensor type as lidar, I am able to do obstacle avoidance in Airsim even the delay is very high, but when I use sensor types as Distance sensors I am not able to do the obstacle avoidance, it is not working, I used Distance sensors because according the documentation range finders in ardupilot is distance sensors in Airsim, I have done obstacle avoidance in an actual drone by setting the range finder parameters in ardupilot, can someone help me doing this?

**Comment by ACanda1** [comment]
What versions of ArduPilot and AirSim are you using to get the LIDAR to work?
